confluent_kafka/__init__.py

```diff
@@ -1,9 +1,29 @@
-__all__ = ['cimpl', 'admin', 'avro', 'kafkatest']
-from .cimpl import (Consumer,  # noqa
+#!/usr/bin/env python
```

Comment:
why does this file need to be executable?

Reply:
reference? why? seems of such marginal benefit it's better no to have it. don't understand why it would be best practice. I just looked at the code of the 3 of the most popular libraries for python, and none of them do this.